docs-devsite/auth.auth.md

```diff
@@ -37,6 +37,7 @@ export interface Auth
 
 |  Method | Description |
 |  --- | --- |
+|  [authStateReady()](./auth.auth.md#authauthstateready) | return a promise that resolves immediately when the initial auth state is settled. The current user might be a valid user, or null if there is no user signed in currently. |
```

Comment:
I think this should this be "[authStateReady()](./auth.auth.md#authauthstateready) | returns" with an "s."

Reply:
Also, there's something off in the sentence, "The current user might be a valid user, or null if there is no user signed in currently."

Are there two possible return values, `valid` and null?  Or does it return a username?  Suggest rephrasing and using backticks if you mention literal values.